src/MarchingCubes.jl

```diff
@@ -86,7 +88,9 @@ struct MC{F,I}
             Ref(x),
             Ref(y),
             Ref(z),
+            normal_sign,
         )
+        m.normal_sign ∈ (-1, +1) || throw(ArgumentError("`normal_sign` should be either -1 or +1")) 
```

Comment:
This can be moved before the `m = ...` construct for early check.

```suggestion
        normal_sign ∈ (-1, +1) || throw(ArgumentError("`normal_sign` should be either -1 or +1")) 
```